tests/unit/mail_test.sh

```diff
@@ -10,6 +10,9 @@ function oneTimeSetUp()
   declare -gr FAKE_KERNEL="$FAKE_GIT/fake_kernel/"
   declare -ga test_config_opts=('test0' 'test1' 'test2' 'user.name' 'sendemail.smtpuser')
 
+  declare -gr FAKE_PATCHES_DIR='fake_patches'
+  declare -gr FAKE_PATCH_FILE="${FAKE_PATCHES_DIR}/fake_patch_file"
```

Comment:
I know that because of the `cd` statement below, we will eventually be at `FAKE_GIT`, and it will be safe to do what you did:

```bash
mkdir "./${FAKE_PATCHES_DIR}"
touch "./${FAKE_PATCH_FILE}"
```

However, I think the cleanest way is to define explicitly as `FAKE_PATCHES_DIR=${FAKE_GIT}/fake_patches` and make the statements below simply:

```bash
  mkdir "$FAKE_PATCHES_DIR"
  touch "$FAKE_PATCH_FILE"
```

this way we don't depend on relative paths which can induce errors that will be hard to debug.